core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -401,7 +401,8 @@ class BrokerServer(
         shareFetchSessionCache,
         config.shareGroupRecordLockDurationMs,
         config.shareGroupDeliveryCountLimit,
-        config.shareGroupPartitionMaxRecordLocks
+        config.shareGroupPartitionMaxRecordLocks,
+        config.ShareGroupPersisterClassName
```

Comment:
I would imagine `ShareGroupPersisterClassName` should start with a lower-case S.